.github/workflows/deploy.yml

```
@@ -1,46 +1,31 @@
     name: Deploying 🚀 Clima
     runs-on: ubuntu-latest
-    if: "contains(github.event.head_commit.message, 'bump version')"
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
 
-      - name: Setup python
-        uses: actions/setup-python@v4
+      - id: auth
+        uses: google-github-actions/auth@v1
```

actionlint (1.7.7)

the runner of "google-github-actions/auth@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)